packages/wallets/src/components/Base/WalletPasswordField/WalletPasswordField.tsx

```diff
@@ -20,7 +68,21 @@ const WalletPasswordField: React.FC<WalletPasswordFieldProps> = ({
     const [isPasswordVisible, setIsPasswordVisible] = useState(false);
     const [isTouched, setIsTouched] = useState(false);
 
-    const { errorMessage, score } = validatePassword(password);
+    const zxcvbn = useRef<any>();
+
+    const validationResult = zxcvbn.current && validatePassword(zxcvbn.current, password);
+    const errorMessage = validationResult?.errorMessage;
+    const score = validationResult?.score;
+
+    useEffect(() => {
+        async function loadLibrary() {
+            const lib = await import('@contentpass/zxcvbn');
```

Comment:
I thought we are using `@zxcvbn-ts`

Reply:
Looks like you're right Farhan. Searching through the code I can see we are using:
- zxcvbn
- zxcvbn-ts
- @contentpass/zxcvbn

I really don't think we should be using any of those three.

Reply:
We could use an entropy check, like:
https://www.npmjs.com/package/fast-password-entropy

Reply:
will attempt to use react-lazy as it seems that it might also be useful for other cases

Reply:
updated with react-lazy @markwylde-deriv